content/id/docs/concepts/scheduling/kube-scheduler.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Penjadwal Kubernetes
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+Dalam Kubernetes, _scheduling_ merujuk dalam hal untuk memastikan 
+{{< glossary_tooltip text="Pods" term_id="pod" >}} mendapatkan
+{{< glossary_tooltip text="Nodes" term_id="node" >}} sehingga
+{{< glossary_tooltip term_id="kubelet" >}} dapat menjalankannya.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Ikhtisar Scheduling {#scheduling}
```

Comment:
Will you use scheduling or penjadwalan here?

Reply:
I think better penjadwalan, I just worried about the hashtag. But I know that I need to change the hashtag also.